### DISCUSSION TITLE
Future Feature : Terminology (LOINC - Vital sign, Physical Examination, Education)

### DISCUSSION THREAD

**Discussion by ivanwilliammd** [ivanwilliammd/satusehat-integration#127] [Ideas]
**Is your feature request related to a problem? Please describe.**
Requirement terminology Rawat Jalan Fase 2, Fase 5 : 
- LOINC

**Describe the solution you'd like**
Data source will be provided - DM me if you want to pick up this feature

**Describe alternatives you've considered**
Create a new 1 class Model object for LOINC will be created

**Additional context**
Required in SATUSEHAT in general, focused use subset instead of full dictionary data.
Enumerator / array list is not recommended due to the volatility

CC: @YogiPristiawan

**Comment by yogip07** [comment]
ini maksudnya issue khusus untuk pengiriman uji laboratorium pak? lalu kenapa tertulis vital-sign juga?

**Comment by ivanwilliammd** [comment]
> ini maksudnya issue khusus untuk pengiriman uji laboratorium pak? lalu kenapa tertulis vital-sign juga?

LOINC juga dipakai di pemeriksaan fisik dan vital-sign juga
Karena saat ini di model observation seingat saya masih pakai array key-value pair, ini tidak sustainable

**Comment by yogip07** [comment]
baik pak. Dibuat seperti ICD-10 di module terminology berarti ya? 

oh iya lalu untuk resource Observation pada bagian code perlu dihilangkan untuk key vaue pair nya? lalu bagaimana cara kita validasi value yang valid?

**Comment by ivanwilliammd** [comment]
nanti coba saya push dulu untuk tsv file-nya...
Nanti kita bisa start dari sana untuk : 
- Seeder
- Config service
- Service model

**Comment by yogip07** [comment]
siap pak
